homeassistant/components/homeassistant/system_health.py

```diff
@@ -15,5 +15,10 @@ def async_register(
 async def system_health_info(hass):
     """Get info for the info page."""
     info = await system_info.async_get_system_info(hass)
-    info.pop("hassio")
+
+    # Moved to homeassistant/components/hassio/system_health.py
+    for key in ["hassio", "supervisor", "host_os", "chassis", "docker_version"]:
```

Comment:
Actually wondering here, maybe we should be explicit about what keys we show? Just in case helper/system_info.py changes. That way we can also decide on the order of keys.

Reply:
Was thinking about that as well, so will do that :+1: